plugins/scaffolder-backend-module-bitbucket-cloud/src/actions/bitbucketCloud.examples.ts

```diff
@@ -0,0 +1,132 @@
+    example: yaml.stringify({
+      steps: [
+        {
+          id: 'publish',
+          action: 'publish:bitbucketCloud',
+          name: 'Publish to Bitbucket Cloud',
+          input: {
+            repoUrl:
+              'bitbucket.org?repo=repo&workspace=workspace&project=project',
+          },
+        },
+      ],
+    }),
+  },
+  {
+    description: 'Add a description.',
+    example: yaml.stringify({
+      steps: [
+        {
+          id: 'publish',
+          action: 'publish:bitbucketCloud',
+          name: 'Publish to Bitbucket Cloud',
+          input: {
+            repoUrl:
+              'bitbucket.org?repo=repo&workspace=workspace&project=project',
+            description: 'Initialize a git repository',
+          },
+        },
+      ],
+    }),
+  },
+  {
+    description: 'Change visibility of the repository.',
+    example: yaml.stringify({
+      steps: [
+        {
+          id: 'publish',
+          action: 'publish:bitbucketCloud',
+          name: 'Publish to Bitbucket Cloud',
+          input: {
+            repoUrl:
+              'bitbucket.org?repo=repo&workspace=workspace&project=project',
+            description: 'Initialize a git repository',
+            repoVisibility: 'public',
+          },
+        },
+      ],
+    }),
+  },
+  {
+    description: 'Set the default branch.',
+    example: yaml.stringify({
+      steps: [
+        {
+          id: 'publish',
+          action: 'publish:bitbucketCloud',
+          name: 'Publish to Bitbucket Cloud',
+          input: {
+            repoUrl:
+              'bitbucket.org?repo=repo&workspace=workspace&project=project',
+            description: 'Initialize a git repository',
+            repoVisibility: 'public',
+            defaultBranch: 'main',
+          },
+        },
+      ],
+    }),
+  },
+  {
+    description: 'Specify a source path within the workspace.',
+    example: yaml.stringify({
+      steps: [
+        {
+          id: 'publish',
+          action: 'publish:bitbucketCloud',
+          name: 'Publish to Bitbucket Cloud',
+          input: {
+            repoUrl:
+              'bitbucket.org?repo=repo&workspace=workspace&project=project',
+            description: 'Initialize a git repository',
+            repoVisibility: 'public',
+            defaultBranch: 'main',
+            sourcePath: './repoRoot',
+          },
+        },
+      ],
+    }),
+  },
+  {
+    description: 'Provide an authentication token for Bitbucket.',
+    example: yaml.stringify({
+      steps: [
+        {
+          id: 'publish',
+          action: 'publish:bitbucketCloud',
+          name: 'Publish to Bitbucket Cloud',
+          input: {
+            repoUrl:
+              'bitbucket.org?repo=repo&workspace=workspace&project=project',
+            description: 'Initialize a git repository',
+            repoVisibility: 'public',
+            defaultBranch: 'main',
+            token: 'your-custom-auth-token',
```

Comment:
Perhaps it makes sense to include the repo url picker parameter definition in the example in order to fill out this example a little bit. Thoughts?